examples/neighbors/plot_caching_nearest_neighbors.py

```diff
@@ -3,15 +3,15 @@
 Caching nearest neighbors
 =========================
 
-This examples demonstrates how to precompute the k nearest neighbors before
+This example demonstrates how to precompute the k nearest neighbors before
```

Comment:
In general, even for small stuff, I would suggest top open a separate PR for unrelated clean-up, there is always the chance to generate side-discussions ...